modules/tide_ckeditor/tide_ckeditor.install

```diff
@@ -21,6 +21,8 @@ function tide_ckeditor_update_9001() {
   $role_ids = ['editor', 'approver', 'site_admin', 'contributor'];
   foreach ($role_ids as $role_id) {
     $role = Role::load($role_id);
-    $role->grantPermission('use text format advanced')->save();
+    if ($role) {
+      $role->grantPermission('use text format advanced')->save();
```

Comment:
contributor role does not exist for all projects, that is why this needs a check before assigning the permission, so that it does not throw null exception.